
Check if daemon is runnnig before stopping it and don't fail if not - consistency change, this is what all other freetz rc.scripts do.

--- root/usr/lib/callmonitor/applets/rc.callmonitor.sh
+++ root/usr/lib/callmonitor/applets/rc.callmonitor.sh
@@ -25,8 +25,12 @@
 }
 stop_daemon() {
 echo -n "Stopping $DAEMON ... "
+if ! is_running; then
+echo 'not running.'
+else
 "$DAEMON" -s
 check_status
+fi
 }
 else
 start_daemon() {
